.circleci/config.yml

```diff
@@ -38,6 +38,15 @@ jobs:
           name: run the actual tests
           command: |
             mvn --batch-mode clean install -Punit-tests
+      - run:
+          name: run the non-confidential integration tests
+          command: |
+            # Adding all "normal" certs into this local one that has the Hoverfly cert (instead of adding Hoverfly cert to the global one) 
```

Comment:
I think you mentioned this verbally in the demo, but for posterity, code comment on the downside of adding the hoverfly cert globally